data/gegno/power.txt

```diff
@@ -0,0 +1,119 @@
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description "Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage."
+
+outfit "R04 Crusade Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description "As the largest battery mounted on Gegno ships, the Crusade Battery supplies almost enough power as a small generator, enough to support the massive warships of both the Scin and the Vi. However, they fall under the capacity of similarly-sized human batteries due to inefficient handling of their scale."
+
+outfit "C3 Scrap Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 54000
+	mass 25
+	thumbnail outfit/vicoretiny
+	"energy generation" 1.4
+	"heat generation" 2.2
+	"outfit space" -25
+	description "The Scrap Cell is a small radiothermal generator that produces energy generation in a pinch that abusing the Gegno's slightly hotter running tech. Its output is very limited, and is only used in tandem with the Gegno's smaller-sized systems."
+
+outfit "C7 Brawl Core"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 68100
+	mass 39
+	thumbnail outfit/vicoresmall
+	"energy generation" 2.5
+	"heat generation" 3.9
+	"outfit space" -39
+	description "C7 Brawl Cores are a slight deviation from the other C class cores as it is set upright-forward. It is a more rudimentary type of radiothermal reactor than the C3, but nevertheless produces the necessary power for the ships that typically carry one."
+
+outfit "C17 Warzone Core"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "C7 Warzone Cores"
+	cost 2472000
+	mass 87
+	thumbnail outfit/vicoremed
+	"energy generation" 14.1
+	"heat generation" 38.3
+	"outfit space" -87
+	description "As did humans, the Gegno began developing nuclear technology once their cities grew massive enough to require it. The Warzone Core is a medium-sized nuclear fission reactor that is a suitable power source for most of the Gegno's large ships, though it looks even more discomforting than a human one."
+
+outfit "C27 Campaign Core"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "C7 Campaign Cores"
+	cost 8145000
+	mass 145
+	thumbnail outfit/vicorelarge
+	"energy generation" 22.3
+	"heat generation" 62.9
+	"outfit space" -145
+	description "In order to power the largest of vessels along with their power-guzzling systems and engines, the Gegno designed the Campaign Fusion Reactor from earlier prototypes during their colony ship era. It's still used to this day and is as efficient as one warship needs it to be, but just looking at it still raises concerns."
```

Comment:
`as efficient as one warship needs it to be`
What does this mean? Which one warship? Is it possible to share a reactor between more than one warship?

Reply:
How would this sound? Changing "It's still used to this day and is as efficient as one warship needs it to be,", to:

"....to this day and is efficient enough to satisfy the requirements of the largest of Gegno warships, but...."

Reply:
Sounds good.